src/effects/VST3/VST3Utils.h

```diff
@@ -52,7 +56,73 @@ class VST3Utils final
    //a human-readable form.
    static wxString MakeAutomationParameterKey(const Steinberg::Vst::ParameterInfo& info);
 
-   //Attempts to extract a unique VST3 parameter id from the key.
-   //Returns true on success.
-   static bool ParseAutomationParameterKey(const wxString& key, Steinberg::Vst::ParamID& paramId);
 };
+
+struct EffectSettings;
+
+struct VST3EffectSettings
+{
+   std::unordered_map<Steinberg::Vst::ParamID, Steinberg::Vst::ParamValue> mValues;
```

Comment:
The commit that introduces this line needs `#include <unordered_map>` to compile on macOS

Reply:
thanks, added